### PR TITLE
mplayer 1.3.0

### PR DESCRIPTION
**Pull request by makigumo**

### CODE REVIEW

File: Library/Formula/mplayer.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@ def install
 +++ b/configure
 @@ -1517,8 +1517,6 @@ if test -e ffmpeg/mp_auto_pull ; then
  fi
- 
+
```
Comment (DomT4) [inline Library/Formula/mplayer.rb]: Careful of tweaking patch spacing like this. Has the potential to cause fun little breakage depending on user configuration.

Comment (makigumo) [inline Library/Formula/mplayer.rb]: You are right! I will fix that immediately.

Comment (DomT4) [inline Library/Formula/mplayer.rb]: Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/09d03cd9be50c090b2a002d1da36faa469709640. Thank you for another contribution to Homebrew @makigumo! :smiley_cat:

**Pull request closed without merging**